projects/angular-contenteditable-accessor/src/lib/contenteditable-value-accessor.ts

```diff
@@ -151,9 +146,14 @@ export class ContenteditableValueAccessor
      * null is treated as empty string to prevent IE11 outputting 'null',
      * also single <br> is replaced with empty string when passed to the control
      */
-    private static processValue(value: unknown): string {
+    private processValue(value: unknown): string {
```

Comment:
Shouldn't it return SafeHtml?

Reply:
<img width="839" alt="image" src="https://github.com/taiga-family/angular-contenteditable-accessor/assets/12021443/fec0bc67-7c71-4b87-ab59-5fec9c8f8b5f">